x/nft/internal/conv/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package conv provides internal functions for convertions and data manipulation
```

Comment:
Maybe this should be under a `unsafe` package of the SDK, to that is public and other modules, like NFT, can use it, without having to copy the whole lib.

Reply:
id ask what performance does this actually bring. in some cases its useful but for something that isn't don't often I don't think we need to do this unsafe tactic

Reply:
I think this actually goes away with collections. since store keys, where this is used, wont have it anymore. Id say lets leave as is and figure out how to remove it